README.md

```diff
@@ -60,21 +60,26 @@ longer track you using cookies or JavaScript. Your subscriptions and history are
 * Show/hide functionality or elements within the app using the distraction free settings
 * View channel community posts
 
-### Browser Extension
-FreeTube is supported by the [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect) and [LibRedirect](https://github.com/libredirect/libredirect) extensions, which will allow you to open YouTube links into FreeTube. 
+### Browser Extensions
+The following extensions open YouTube links directly in FreeTube:
 
-> [!IMPORTANT]  
-> You must enable the option within the advanced settings of the extension for it to work.
+- [LibRedirect](https://libredirect.github.io/)
+- [RedirectTube](https://github.com/MStankiewiczOfficial/RedirectTube)
+
+LibRedirect automatically redirect YouTube links to FreeTube. To ensure proper functionality, select FreeTube as Frontend in the Services settings of the extension.
+
+RedirectTube, doesn’t automatically open YouTube links in FreeTube. Instead, it adds buttons to the toolbar and context menu, which you can click to open videos in FreeTube manually.
 
-* Download Privacy Redirect for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/privacy-redirect/) or [Google Chrome](https://chrome.google.com/webstore/detail/privacy-redirect/pmcmeagblkinmogikoikkdjiligflglb).
+- Download LibRedirect from [Mozilla Add-ons](https://addons.mozilla.org/firefox/addon/libredirect/) (for Firefox based-browsers) or [Chrome Web Store](https://libredirect.github.io/download_chromium.html) (for Chrome and Chromium-based browsers).
 
-* Download LibRedirect for [Firefox](https://addons.mozilla.org/firefox/addon/libredirect/) or [Google Chrome](https://libredirect.github.io/download_chromium.html).
+- Download RedirectTube from [Mozilla Add-ons](https://addons.mozilla.org/firefox/addon/redirecttube/) (for Firefox based-browsers).
 
 > [!NOTE]
-> This extension does not work on Linux portable builds!
->
 > If you have issues with the extension working with FreeTube, please create an issue in this repository instead of the extension repository.
 
+> [!WARNING]
```

Comment:
The statement "These extensions do not work on Linux portable builds!" is more of a note than a warning.

A warning usually suggests a potential risk or problem that could have significant consequences if not heeded. In contrast, a note typically provides important information that users should be aware of, but it doesn't necessarily imply a risk or danger.

In this case, the statement is informing users about a limitation or compatibility issue, without implying a serious risk or adverse consequence.